addons/mail/static/src/xml/composer.xml

```diff
@@ -14,17 +14,18 @@
              </div>
          </t>
          <div t-attf-class="o_composer #{widget.extended ? 'o_extended_composer' : ''}">
-             <div class="o_composer_input">
-                 <textarea class="o_composer_text_field" tabindex="2" placeholder="Write something..."/>
-             </div>
-             <t t-if="!widget.extended">
-                 <div class="btn-group o_composer_buttons">
-                    <button tabindex="3" class="btn btn-sm btn-primary o_composer_button_send" type="button"><t t-esc="widget.options.send_text"/></button>
+             <!-- TODO: Use real user avatar -->
```

Comment:
let say that it's for all the developers assigned to this project...
mmm..  wait...  ^^